lib/api/programming/index.js

```diff
@@ -118,5 +118,6 @@ export default {
     programWithHexFile,
     programWithHexString,
     getFamily,
+    getVersionInfo: probe.getVersionInfo,
```

Comment:
This is ok for now, but the `getVersionInfo()` will only work with the connectivity firmware, so we should not probably expose this function in the released version. I also suspect that this will not be available anymore in pc-nrfjprog-js when @CoqRogue has completed the rewrite.